WeakAuras/WeakAuras.lua

```diff
@@ -1445,6 +1451,7 @@ loadFrame:RegisterEvent("UNIT_EXITED_VEHICLE");
 loadFrame:RegisterEvent("SPELLS_CHANGED");
 loadFrame:RegisterEvent("GROUP_JOINED");
 loadFrame:RegisterEvent("GROUP_LEFT");
+loadFrame:RegisterEvent("UNIT_AURA");
```

Comment:
Nope. ScanForLoads should not be called for each UNIT_AURA. That's far to expensive and comes in far too often in a raid.